.github/CODEOWNERS

```diff
@@ -11,10 +11,12 @@
 /licenses/                @dfinity/idx
 /bin/ict                  @dfinity/idx
 /bin/                     @dfinity/idx
-/bin/afl_test.sh          @dfinity/product-security
-/bin/afl_wrapper.sh       @dfinity/product-security
+/bin/afl_corpus_min.sh     @dfinity/product-security
```

Comment:
What about moving all fuzzing scripts into a dedicated `bin/fuzzing` directory. That way you need a single ownership rule and it'
s immediately clear which scripts are and aren't used for fuzzing.